internal/controller/postgrescluster/controller.go

```diff
@@ -183,8 +183,12 @@ func (r *Reconciler) Reconcile(
 		if !equality.Semantic.DeepEqual(before.Status, cluster.Status) {
 			// NOTE(cbandy): Kubernetes prior to v1.16.10 and v1.17.6 does not track
 			// managed fields on the status subresource: https://issue.k8s.io/88901
+
+			o := &client.SubResourcePatchOptions{}
+			o.FieldManager = string(r.Owner)
```

Comment:
I'm not sure if this is correct but it is what the API expects. I looked at how Crunchy did it but currently, they still use controller-runtime `v0.12.3`.

Reply:
I think it's OK